html/pfappserver/lib/pfappserver/PacketFence/Controller/Config/Pfmon.pm

```diff
@@ -0,0 +1,98 @@
+package pfappserver::PacketFence::Controller::Config::Pfmon;
+
+=head1 NAME
+
+pfappserver::PacketFence::Controller::Config::Pfmon - Catalyst Controller
+
+=head1 DESCRIPTION
+
+Controller for admin roles management.
+
+=cut
+
+use HTTP::Status qw(:constants is_error is_success);
+use Moose;  # automatically turns on strict and warnings
+use namespace::autoclean;
+
+use pfappserver::Form::Config::Switch;
+use pf::config::cached;
+
+BEGIN {
+    extends 'pfappserver::Base::Controller';
+    with 'pfappserver::Base::Controller::Crud::Config';
+    with 'pfappserver::Base::Controller::Crud::Config::Clone';
+}
+
+__PACKAGE__->config(
+    action => {
+        # Reconfigure the object action from pfappserver::Base::Controller::Crud
+        object => { Chained => '/', PathPart => 'config/pfmon', CaptureArgs => 1 },
+        # Configure access rights
+        view   => { AdminRole => 'PFMON_READ' },
+        list   => { AdminRole => 'PFMON_READ' },
+        create => { AdminRole => 'PFMON_READ' },
```

Comment:
Could you group them together and add a comment on those to explain it as when looking at it at first hand it might lead to think they are actually valid actions